src/components/SideBar.tsx

```diff
@@ -1,78 +1,262 @@
+  audioArea: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    textAlign: 'center'
+  },
+  audioPictureContainer: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    height: 130,
+    width: 130,
+    border: 'solid rgb(208, 53, 30) 1px',
+    borderRadius: '130px',
+    '@media (max-width: 450px)': {
+      width: 60,
+      height: 60
+    }
+  },
+  playButtonArea: {
+    display: 'flex',
+    flexDirection: 'row'
+  },
+  audioPlayButton: {
+    padding: 5,
+    cursor: 'pointer'
+  },
+  sideBarContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    background: '#1a1919',
+    color: 'white',
+    '@media (max-width: 959px)': {
+      position: 'fixed',
+      bottom: 0,
+      left: 0,
+      width: '100%',
+      minHeight: '60px'
+    },
+    '@media (min-width: 960px)': {
+      height: '100%'
+    } 
   },
-  playArea: {
+  userPicture: {
+    height: 120,
+    width: 120,
+    borderRadius: '50%',
+    '@media (max-width: 959px)': {
+      height: 60,
+      width: 60
+    },
+  },
+  userPictureContainer: {
     textAlign: 'center',
-    bottom: 0,
-    paddingBottom: 20,
+    '@media (max-width: 959px)': {
+      position: 'absolute',
+      left: 0,
+      top: 0
+    },
   },
-  beatPictureContainer: {
-    width: 200,
-    borderColor: 'red',
+  userStatContainer: {
+    
+    '@media (max-width: 959px)': {
+      display: 'none'
+    },
   }
 }));
 
-const SideBar: React.FC = () => {
+
+const SideBar: React.FC<SideBarProps> = ({...props}) => {
   const classes = useStyles();
-  
-  return(
-    <div className={classes.sidebar}>
-      <div className={classes.userStatus}>
-        Log out
-        <span> Put icon Here</span>
-      </div>
-      <div className={classes.header}>
-        <img alt='' className={classes.picture} src="https://upload.wikimedia.org/wikipedia/commons/thumb/3/37/African_Bush_Elephant.jpg/440px-African_Bush_Elephant.jpg"></img>
-        <div>Your name here</div>
-      </div>
+  const [audioArray, setAudioArray] = useState([] as AudioObject[]);
+
+  const { logout } = useAuth0(); 
 
-      <div className={classes.stat}>
-        <div>
-          Beats: 155
+  useEffect(() => {
+    console.log("props.id : " + props.id);
+    console.log("audioArrayLength : " + audioArray.length);
+    if (props.id >= 1 && props.id <= 5) {
+      setAudioArray([
+        {
+          'imageUrl': 'https://qph.fs.quoracdn.net/main-qimg-70d48d9e6d598aa364c13ef739b489d4',
+          'audioUrl': 'http://www.hubharp.com/web_sound/WalloonLilliShort.mp3',
+          'title': 'audio 1 title',
+          'authorName': 'author 1'
+        },
+        {
+          'imageUrl': 'https://qph.fs.quoracdn.net/main-qimg-70d48d9e6d598aa364c13ef739b489d4',
+          'audioUrl': 'http://www.hubharp.com/web_sound/BachGavotteShort.mp3',
+          'title': 'audio 2 title',
+          'authorName': 'author 2'
+        }
```

Comment:
Test data should be in own json file. Makes it easier to change values for testing, especially as this grows over time